subliminal/providers/_agent_list.py

```diff
@@ -0,0 +1,783 @@
+FIRST_THOUSAND_OR_SO_USER_AGENTS = [
```

Comment:
i'd just get a really generic Chrome on Windows User-Agent, can't see a reason to have *this many* of them

Reply:
I put  a list of 6 user-agents in the `addic7ed.py` file.